Lib/defconQt/featureTextEditor.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, font=None, parent=None):
         self.editor.modificationChanged.connect(self.setWindowModified)
 
     def setWindowTitle(self, title="Font Features", font=None):
+        title = self.tr("Font Features")
```

Comment:
And please use any of the newer Python formatting features, the `printf` style formatting in Python is not good for l10n.